force-app/main/default/flows/STO_Reserve_and_Complete.flow-meta.xml

```diff
@@ -0,0 +1,636 @@
+                <leftValueReference>actionType</leftValueReference>
+                <operator>EqualTo</operator>
+                <rightValue>
+                    <stringValue>Complete</stringValue>
+                </rightValue>
+            </conditions>
+            <connector>
+                <targetReference>Already_Complete</targetReference>
+            </connector>
+            <label>Complete Path</label>
+        </rules>
+        <rules>
+            <name>Reserve</name>
+            <conditionLogic>and</conditionLogic>
+            <conditions>
+                <leftValueReference>actionType</leftValueReference>
+                <operator>EqualTo</operator>
+                <rightValue>
+                    <stringValue>Reserve</stringValue>
+                </rightValue>
+            </conditions>
+            <connector>
+                <targetReference>Already_Reserved</targetReference>
+            </connector>
+            <label>Reserve Path</label>
+        </rules>
+    </decisions>
+    <description>This will wil either reserve a case to a Saksbehandler or Ferdigstille it, depending on which event that triggered this flow.</description>
+    <environments>Default</environments>
+    <interviewLabel>STO Reserve and Complete {!$Flow.CurrentDateTime}</interviewLabel>
+    <label>STO Reserve and Complete</label>
+    <processMetadataValues>
+        <name>BuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>CanvasMode</name>
+        <value>
+            <stringValue>AUTO_LAYOUT_CANVAS</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>OriginBuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processType>Flow</processType>
+    <recordLookups>
+        <name>Get_Case</name>
+        <label>Get Case</label>
+        <locationX>941</locationX>
+        <locationY>242</locationY>
+        <assignNullValuesIfNoRecordsFound>false</assignNullValuesIfNoRecordsFound>
+        <connector>
+            <targetReference>Which_Action</targetReference>
+        </connector>
+        <filterLogic>and</filterLogic>
+        <filters>
+            <field>Id</field>
+            <operator>EqualTo</operator>
+            <value>
+                <elementReference>recordId</elementReference>
+            </value>
+        </filters>
+        <getFirstRecordOnly>true</getFirstRecordOnly>
+        <object>Case</object>
+        <storeOutputAutomatically>true</storeOutputAutomatically>
+    </recordLookups>
+    <recordLookups>
+        <name>Get_Thread</name>
+        <label>Get Thread</label>
+        <locationX>677</locationX>
+        <locationY>242</locationY>
+        <assignNullValuesIfNoRecordsFound>false</assignNullValuesIfNoRecordsFound>
```

Comment:
Du henter ut Tråden her, men det trengs ikke, vi bryr oss ikke om noen av feltene på tråden, bare casen. Flowen kommer også til å feile i dette tilfelle, fordi vi henter aldri en case for decisionen 😄.